orangecontrib/text/widgets/owwordcloud.py

```diff
@@ -44,6 +44,7 @@ class OWWordCloud(widget.OWWidget):
 
     class Warning(widget.OWWidget.Warning):
         topic_precedence = widget.Msg('Input signal Topic takes priority over Corpus')
+        empty_corpus = widget.Msg('Empty Corpus')
```

Comment:
"Input Corpus is empty."

Is there any other widget that does this? @astaric Is this desirable?